### ISSUE TITLE
t/mojolicious/websocket_lite_app.t fails on Windows

### ISSUE THREAD

**Issue by chorny** [kraih/mojo#1011]
- Mojolicious version: 7.09
- Perl version: 5.14.0, 5.24.1-RC4
- Operating system: Windows XP

```
Use of uninitialized value in numeric ge (>=) at C:\strawberry514\cpan\build\Mojolicious-7.09-0\blib\lib/Mojo/Server/Daemon.pm line 98.
# WebSocket closed with status 1006
not ok 56 - WebSocket closed with status 1009
#   Failed test 'WebSocket closed with status 1009'
#   at t/mojolicious/websocket_lite_app.t line 163.
```

Also on 7.08, test passes on 7.07.

**Comment by kraih** [comment]
We've got a lot of success reports from Windows users, so this appears to be a very rare problem and will be almost impossible for us to track down.

**Comment by kraih** [comment]
I've seen reports like this a few times now, but so far we have no way to replicate the problem.

**Comment by CandyAngel** [comment]
I can't replicate the failure of the test, "# 64-bit binary message (too large for server)" passes, but I do also get the error:

    Use of uninitialized value in numeric ge (>=) at C:\strawberry514\cpan\build\Mojolicious-7.09-0\blib\lib/Mojo/Server/Daemon.pm line 98.

I've done some investigation and it seems that the $tx's connection is removed from Mojo::Server::Daemon after the first request, but then that $tx receives another request. However, as $self->{connections}{$id} has already been deleted, when it checks $c->{requests} (now undefined) against $self->max_requests in Mojo::Server::Daemon::_build_tx(), the above error is caused.

This only seems to happen on this test (the others only receive one request), so perhaps something between the test suite and the test server is retrying?

## TL;DR:
```
# REQUEST  4b66549067f8262497fcf95ce1fff5df
# FINISH:  4b66549067f8262497fcf95ce1fff5df
ok 54 - WebSocket handshake with /echo
ok 55 - send message
# CLOSE: 4b66549067f8262497fcf95ce1fff5df
# REQUEST  4b66549067f8262497fcf95ce1fff5df
Use of uninitialized value in numeric ge (>=) at P:\tmp\strawberry-perl-5.24.0.1-64bit-PDL\cpan\build\Mojolicious-7.10-J0Heya\lib/Mojo/Server/Daemon.pm line 102.
# CLOSE: 4b66549067f8262497fcf95ce1fff5df
ok 56 - WebSocket closed with status 1009
```
## Showing other tests only make one request:
```
# REQUEST  12a94c01176cb0500c7f9f32f3f1acc6
# FINISH:  12a94c01176cb0500c7f9f32f3f1acc6
ok 14 - WebSocket handshake with /echo
ok 15 - Sec-WebSocket-Accept: I+x5C3/LJxrmDrWw42nMP4pCSes=
ok 16 - Sec-WebSocket-Protocol:
ok 17 - send message
ok 18 - message received
ok 19 - exact match for message
# CLOSE: 12a94c01176cb0500c7f9f32f3f1acc6
# CLOSE: 12a94c01176cb0500c7f9f32f3f1acc6
ok 20 - closed WebSocket
ok 21 - right "DNT" value
ok 22 - right "Sec-WebSocket-Protocol" value
# REQUEST  381b7f0d3db2e057f3f9dd55f065c178
# FINISH:  381b7f0d3db2e057f3f9dd55f065c178
ok 23 - WebSocket handshake with /echo
ok 24 - send message
ok 25 - message received
ok 26 - exact match for message
ok 27 - send message
ok 28 - message received
ok 29 - no match for message
# CLOSE: 381b7f0d3db2e057f3f9dd55f065c178
# CLOSE: 381b7f0d3db2e057f3f9dd55f065c178
ok 30 - closed WebSocket
# REQUEST  ad6719ab859e68c648e9497cb2bb6576
# FINISH:  ad6719ab859e68c648e9497cb2bb6576
ok 31 - WebSocket handshake with /echo
ok 32 - send message
ok 33 - send message
ok 34 - send message
ok 35 - message received
ok 36 - exact match for message
# CLOSE: ad6719ab859e68c648e9497cb2bb6576
# CLOSE: ad6719ab859e68c648e9497cb2bb6576
ok 37 - closed WebSocket
# REQUEST  2953e9cde3b8dfe2be0ff8d718b83076
# FINISH:  2953e9cde3b8dfe2be0ff8d718b83076
ok 38 - WebSocket handshake with /echo
ok 39 - send message
ok 40 - message received
ok 41 - exact match for message
ok 42 - send message
ok 43 - message received
ok 44 - message is similar
# CLOSE: 2953e9cde3b8dfe2be0ff8d718b83076
# CLOSE: 2953e9cde3b8dfe2be0ff8d718b83076
ok 45 - closed WebSocket
ok 46 - WebSocket closed with status 1000
# REQUEST  fcc57f86c076a18268cc0812711ae40f
# FINISH:  fcc57f86c076a18268cc0812711ae40f
ok 47 - WebSocket handshake with /echo
ok 48 - right size
ok 49 - send message
ok 50 - message received
ok 51 - exact match for message
# CLOSE: fcc57f86c076a18268cc0812711ae40f
# CLOSE: fcc57f86c076a18268cc0812711ae40f
ok 52 - closed WebSocket
ok 53 - WebSocket closed with status 1005
# REQUEST  4b66549067f8262497fcf95ce1fff5df
# FINISH:  4b66549067f8262497fcf95ce1fff5df
ok 54 - WebSocket handshake with /echo
ok 55 - send message
# CLOSE: 4b66549067f8262497fcf95ce1fff5df
# REQUEST  4b66549067f8262497fcf95ce1fff5df
Use of uninitialized value in numeric ge (>=) at P:\tmp\strawberry-perl-5.24.0.1-64bit-PDL\cpan\build\Mojolicious-7.10-J0Heya\lib/Mojo/Server/Daemon.pm line 102.
# CLOSE: 4b66549067f8262497fcf95ce1fff5df
ok 56 - WebSocket closed with status 1009
1..56
ok
All tests successful.
Files=1, Tests=56,  2 wallclock secs ( 0.06 usr +  0.02 sys =  0.08 CPU)
Result: PASS
```

**Comment by jberger** [comment]
This issue seems to have gone stale. Without the ability to replicate I'm going to close this one. Please reopen if new information comes to light.
